imap_processing/tests/ialirt/unit/test_generate_coverage.py

```diff
@@ -32,3 +33,45 @@ def test_generate_coverage(furnish_kernels):
 
     # Coverage duration should be approximately 9 hours.
     assert hours == pytest.approx(9, abs=1)
+
+
+@pytest.mark.external_kernel
+def test_use_outages(furnish_kernels):
+    """
+    Test that outages are properly used.
+    """
+    # Note: tested this code with the Sun and achieved expected
+    # results ~12 hours of coverage from horizon to horizon.
+    kernels = [
+        "naif0012.tls",
+        "pck00011.tpc",
+        "de440s.bsp",
+        "imap_spk_demo.bsp",
+        "earth_1962_240827_2124_combined.bpc",
+    ]
+
+    outages = {
+        "Kiel": [
+            ("2026-09-22T11:50:00.00Z", "2026-09-22T12:10:00Z"),
```

Comment:
So the idea is that the mentioned cron job will look for outages in text files, format those in some kind of way, and then pass that into the `generate_coverage` function?